### PR TITLE
OSDOCS-5392 Updating vSphere 4.10 docs to require vSphere 7

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.10

Issue:
https://issues.redhat.com/browse/OSDOCS-5392

Link to docs preview:
https://bscott-rh.github.io/openshift-docs/OSDOCS-5392/installing/installing_vsphere/preparing-to-install-on-vsphere.html#installation-vsphere-infrastructure_preparing-to-install-on-vsphere
https://bscott-rh.github.io/openshift-docs/OSDOCS-5392/release_notes/ocp-4-10-release-notes.html#ocp-4-10-installation-vsphere-csi
https://bscott-rh.github.io/openshift-docs/OSDOCS-5392/release_notes/ocp-4-10-release-notes.html#ocp-4-10-deprecated-removed-features

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by bscott-rh** [comment]
@sgaoshang can you please take a look at this change for the vSphere version requirements for 4.10? Thank you

**Comment by sgaoshang** [comment]
@bscott-rh lgtm, thanks.

**Comment by bscott-rh** [comment]
Engineering: @rvanderp3 
PM: @racedo 
PX: @sferich888 
Docs: @kalexand-rh 
QE: @sgaoshang (I am requesting a second review because I have updated the 4.10 release notes as well)

Hello all, please take a look at this PR indicating that vSphere 6 and HW version 13 are no longer supported for OCP 4.10. I'm requesting Change Management approval due to changing the supported status of vSphere 6.x in an existing release. I have posted a slack discussion in the Jira showing where this request originated. Thank you

**Comment by kalexand-rh** [comment]
LGTM

File: modules/vmware-csi-driver-reqs.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 To install the vSphere CSI Driver Operator, the following requirements must be met:
 
-* VMware vSphere version 6.7U3 or later
+* VMware vSphere version 7.0.1 or later
```
Comment (julienlim) [inline modules/vmware-csi-driver-reqs.adoc:27]: Should we want to mention that VMware vSphere version 7.0 Update 1 is now deprecated, and that customers should use 7.0 update 2 or later?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -1445,17 +1443,17 @@ In the table, features are marked with the following statuses:
 |vSphere 6.7 Update 2 or earlier
```
Comment (julienlim) [inline release_notes/ocp-4-10-release-notes.adoc:1443]: vSphere 7.0 update 1 - we should add that it's deprecated in this table for OpenShift 4.10.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -1445,17 +1443,17 @@ In the table, features are marked with the following statuses:
...
-|DEP
+|REM
 
 |Virtual hardware version 13
 |GA
 |DEP
-|DEP
+|REM
 
 |VMware ESXi 6.7 Update 3 or earlier
```
Comment (julienlim) [inline release_notes/ocp-4-10-release-notes.adoc:1453]: VMware ESXi 7.0 update 1 - we should add that it's deprecated in this table for OpenShift 4.10.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -1445,17 +1443,17 @@ In the table, features are marked with the following statuses:
 |vSphere 6.7 Update 2 or earlier
 |GA
 |DEP
-|DEP
+|REM
 
 |Virtual hardware version 13
```
Comment (julienlim) [inline release_notes/ocp-4-10-release-notes.adoc:1448]: Isn't Virtual hardware version 14 also deprecated as well?

Comment (bscott-rh) [reply release_notes/ocp-4-10-release-notes.adoc:1453]: @sgaoshang I've updated the PR with some feedback from Ju. Can you take another look and confirm whether HW version 14 should also be added as deprecated? Thank you

Comment (sgaoshang) [reply release_notes/ocp-4-10-release-notes.adoc:1453]: @bscott-rh Yes, I think so. According to CSI driver requirement, we should use HW version 15 from OCP 4.9, details see https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-vsphere-and-virtual-hardware-version-deprecations @duanwei33 pls correct me if I'm wrong, thanks.

Comment (duanwei33) [reply release_notes/ocp-4-10-release-notes.adoc:1453]: hmm, I remember we had discussion for HW version 14, seems we never announce we support 14, so we did not mention that in our deprecated part previously.  
I think it is safe to keep current status(?) Otherwides, we need to update all doc according to this topic.

**Comment by bscott-rh** [comment]
4.10 documentation is no longer being updated.

**Pull request closed without merging**